src/js/views/maps/LayerDetailsView.js

```diff
@@ -217,11 +217,18 @@ define([
           // Render each section in the Details panel
           this.renderedSections = _.clone(this.sections);
 
+          // Remove and do not render opacity section if showOpacitySlider is false
+          if (model.get("showOpacitySlider") === false) {
+            this.renderedSections = this.renderedSections.filter(
+              (item) => item.label !== "Opacity",
+            );
+          }
+
           this.renderedSections.forEach(function (section) {
-            var detailSection = new LayerDetailView({
+            const detailSection = new LayerDetailView({
```

Comment:
I implemented the changes on line 228, line 231 following the reviewdog suggestions.